api/tests/routes/backoffice/offers_test.py

```diff
@@ -908,3 +909,140 @@ def test_batch_reject_offers(self, legit_user, authenticated_client):
             assert offer.isActive is False
             assert offer.lastValidationType is OfferValidationType.MANUAL
             assert offer.validation is offers_models.OfferValidationStatus.REJECTED
+            assert offer.lastValidationAuthor == legit_user
+
+
+class GetDetailOfferTest(GetEndpointHelper):
```

Comment:
Pour coller avec le nom de l'endpoint : 
```suggestion
class GetOfferDetailsTest(GetEndpointHelper):
```